compiler/modules/types/table.bal

```diff
@@ -10,5 +11,5 @@ final BasicTypeOps tableOps = {
     intersect: bddSubtypeIntersect,
     diff: bddSubtypeDiff,
     complement: bddSubtypeComplement,
-    isEmpty: mappingSubtypeIsEmpty
+    isEmpty: listSubtypeIsEmpty
```

Comment:
I thought we decided this wouldn't work, because the TOP would be wrong.  We need it to be `map<any|error>[]` not `(any|error)[]`.